infrastructure/cdn-in-a-box/ort/traffic_ops_ort/to_api.py

```diff
@@ -156,31 +179,21 @@ def getMyConfigFiles(self, conf:Configuration) -> typing.List[dict]:
 		:raises ConnectionError: when something goes wrong communicating with Traffic Ops
 		"""
 		logging.info("Fetching list of configuration files from Traffic Ops")
+		atstccfg_cmd = self.atstccfg_cmd
+		if conf.mode is Configuration.Modes.REVALIDATE:
+			atstccfg_cmd = self.atstccfg_cmd + ["--revalidate-only"]
 		for _ in range(self.retries):
 			try:
-				# The API function decorator confuses pylint into thinking this doesn't return
-				#pylint: disable=E1111
-				myFiles = self.get_server_config_files(host_name=self.hostname)
-				#pylint: enable=E1111
-				break
-			except (InvalidJSONError, LoginError, OperationError, RequestException) as e:
+				proc = subprocess.run(atstccfg_cmd, stderr=subprocess.PIPE, stdout=subprocess.PIPE)
+				logging.debug("Raw response: %s", proc.stdout.decode())
+				if proc.stderr.decode():
+					logging.error(proc.stderr.decode())
```

Comment:
I just did that because I can't know ahead of time what kinds of messages are in stderr. Since there could be errors, I bring it up to the error level. The only way to fix that would be to iterate the lines of output, parse them and then log each one to the appropriate level, which I thought was a waste of time (and also possibly fruitless, since atstccfg is supposedly meant to be a black box).

Reply:
Agreed, naively printing it sounds preferable in this case